.github/workflows/test-package.yml

```diff
@@ -3,24 +3,23 @@ name: Dart CI
 on:
   # Run on PRs and pushes to the default branch.
   push:
-    branches: [ master ]
+    branches: [ dart-dev, master ]
```

Comment:
Do we really want to run CI on this branch?

Reply:
We have decided to keep the `dart-dev` branch for features that are only available on the dev SDK.

We can remove this, but I think it's better to keep it, or we'd have to change this line every time we pull changes between `maser` and `dart-dev` branch.

Reply:
We want to have a solution for the common situation that:
1) We have a stable version of the package that uses the last Dart stable release and someone makes a PR for that.
2) We have a dev version of the package that uses the last Dart dev release (including new dart:ffi features) and someone makes a PR for that.

So, I think we need two branches.